src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -172,8 +172,7 @@ export default class TelemetryTable extends EventEmitter {
     this.removeTelemetryCollection(keyString);
 
     let sortOptions = this.configuration.getConfiguration().sortOptions;
-    requestOptions.order =
-      sortOptions?.direction ?? (this.telemetryMode === 'performance' ? 'desc' : 'asc');
+    requestOptions.order = sortOptions?.direction ?? 'desc'; // default to descending
```

Comment:
should we extract the string literals to consts somewhere that makes sense? i see we use string literals throughout the table code. it would be nice to clean this up a bit

Reply:
+1 to that. Maybe a map? Doesnt have to be native map either, a object with key/values are good enough

Reply:
good call! I've added a constants file with constants for mode and sort order. I'm sure there's more we could add, but for the purposes of this PR I want to start small.